.devcontainer/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# [Choice] Ruby version (use -bullseye variants on local arm64/Apple Silicon): 3, 3.1, 3.0, 2, 2.7, 2.6, 3-bullseye, 3.1-bullseye, 3.0-bullseye, 2-bullseye, 2.7-bullseye, 2.6-bullseye, 3-buster, 3.1-buster, 3.0-buster, 2-buster, 2.7-buster, 2.6-buster
+ARG VARIANT=3.1-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/ruby:0-${VARIANT}
+
+RUN gem install rails webdrivers 
```

Comment:
We don't want/need to do this, since doing a `bundle install` will do it for us, and get the correct versions.

Reply:
This was just the default ruby dev container stuff, I can remove it if bundle install grabs rails.